addons/mail/static/tests/chatter_tests.js

```diff
@@ -212,13 +214,14 @@ QUnit.test('list activity widget with exception', async function (assert) {
     assert.verifySteps([
         '/web/dataset/search_read',
         '/mail/init_messaging',
+        '/mail/init_messaging_mail_failures',
```

Comment:
I would rather if we didn't `assert.step` every route instead, this is completely irrelevant to this test.